pkg/addons/seaweedfs/seaweedfs.go

```diff
@@ -0,0 +1,208 @@
+package seaweedfs
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/pkg/helpers"
+	"github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/kubeutils"
+	"github.com/replicatedhq/embedded-cluster/pkg/spinner"
+)
+
+const (
+	releaseName = "seaweedfs"
+	namespace   = "seaweedfs"
+)
+
+// Overwritten by -ldflags in Makefile
+var (
+	ChartURL  = "https://url"
+	ChartName = "name"
+	Version   = "v0.0.0"
+)
+
+var helmValues = map[string]interface{}{
+	"master": map[string]interface{}{
+		"replicas": 1,
+		"data": map[string]interface{}{
+			"type": "persistentVolumeClaim",
+			"size": "1Gi",
+		},
+		"logs": map[string]interface{}{
+			"type": "persistentVolumeClaim",
+			"size": "1Gi",
+		},
+		"disableHttp": true,
+	},
+	"volume": map[string]interface{}{
+		"replicas": 1,
+		"dataDirs": []map[string]interface{}{
+			{
+				"name":         "data1",
+				"type":         "persistentVolumeClaim",
+				"size":         "10Gi",
+				"storageClass": "openebs-hostpath",
```

Comment:
Why is storage class only specified for the `volume`? We should either set it for all or remove it for all to use the default.